### PR TITLE
FSI: Remove theta ramp

### PR DESCRIPTION
**Pull request by psakievich**
The theta ramping we implemented was ill-posed and would occasionally generate nan's when computing the angle between a point and the hub-to-blade-root vector.  Fortunately, it is not required at all anymore since we are using the hub deflections as the base deflection.

This PR removes the theta ramping all together. This allows my surrogate mesh case to run fine.

### CODE REVIEW

**Review by marchdf — APPROVED** [review]

**Review by gantech — COMMENTED** [review]
This looks good. Did we test this on the sample case? Does the pitch ramping look ok?

File: src/aero/fsi/FSIturbine.C [hunk 1]
```diff
@@ -240,6 +242,12 @@ fsiTurbine::setup(std::shared_ptr<stk::mesh::BulkData> bulk)
     deflectionRamp_ = &(meta.declare_field<ScalarFieldType>(
       stk::topology::NODE_RANK, "deflection_ramp"));
 
+  distanceToRoot_ = meta.get_field<ScalarFieldType>(
+    stk::topology::NODE_RANK, "distance_to_root");
+  if (distanceToRoot_ == NULL)
+    distanceToRoot_ = &(meta.declare_field<ScalarFieldType>(
```
Comment (gantech) [inline src/aero/fsi/FSIturbine.C]: Is there a way we can add fields that are created only in debug mode?

**Comment by psakievich** [comment]
> This looks good. Did we test this on the sample case? Does the pitch ramping look ok?

Yes the theta ramping didn't really have an affect after we moved to using the hub for the stiff blade deflections.

File: src/aero/fsi/FSIturbine.C (hunk 1 above)
Comment (psakievich) [reply src/aero/fsi/FSIturbine.C]: I can make it so we only use this field if it is requested. I thought I deleted this field though.

**Pull request merged**